<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CSettingsEditorClass</class>
  <widget class="QDialog" name="CSettingsEditorClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>352</width>
        <height>239</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Settings</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
        <widget class="QGroupBox" name="groupBoxApp">
          <property name="title">
            <string>Application</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
              <widget class="QCheckBox" name="checkBoxLoadLast">
                <property name="text">
                  <string>Load last flowsheet on application start</string>
                </property>
                <property name="toolTip">
                  <string>Load last opened flowsheet on application start</string>
                </property>
                
                <property name="whatsThis">
                  <string>Load last opened flowsheet on application start</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                  <widget class="QPushButton" name="pushButtonOpenConfig">
                    <property name="text">
                      <string>Open config file</string>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Open the application-wide configuration file</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Open the application-wide configuration file</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>40</width>
                        <height>20</height>
                      </size>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBoxCache">
          <property name="title">
            <string>Caching</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QLabel" name="label">
                    <property name="text">
                      <string>Cache path</string>
                    </property>
                    <property name="toolTip">
                      <string>Path to store cache files</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Path to store cache files</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="labelWarningCachePath">
                    <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=" color:#ff0000;"&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="lineEditCachePath">
                    <property name="enabled">
                      <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                      <string notr="true">background-color: lightGray</string>
                    </property>
                    <property name="readOnly">
                      <bool>true</bool>
                    </property>
                    <property name="toolTip">
                      <string>Path to store cache files</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Path to store cache files</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                  <widget class="QPushButton" name="pushButtonChangeCache">
                    <property name="text">
                      <string>Change path...</string>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Change cache path</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Change cache path</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="pushButtonClearCache">
                    <property name="text">
                      <string>Clear cache</string>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Delete all cache files</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Delete all cache files</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="horizontalSpacer">
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>40</width>
                        <height>20</height>
                      </size>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QLabel" name="labelWarning">
                    <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=" color:#ff0000;"&gt;*&lt;/span&gt;Will be applied after restart&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>